index.html

```diff
@@ -11,21 +11,121 @@
 
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
-  <![endif]-->
+     <![endif]-->
 </head>
 
 <body>
 
     <div class="container">
       <div class="row">
           <div class="col-sm-12">
-              <h1>Start Here!</h1>
+              <h1>April Resume</h1>
           </div>
       </div>
     </div><!-- container -->
+    <div class="container">
+      <div class="row justify-content-end">
+          <nav class="nav">
+          <a class="nav-link active" href="#">Home</a>
+          <a class="nav-link" href="#">About</a>
+          <a class="nav-link" href="#">Portfolio</a>
+          <a class="nav-link" href="#">Contact</a>
+          </nav>
+      </div>
+      <div class="row">
+          <div class="col-md-12">
+          <div class="jumbotron">
+              <img src= "img/me.jpg" class="img-fluid" alt="Responsive image" height="100" width="100">
```

Comment:
Unfortunately this image didn't end up responsive. Change the height and width values to percentages and you should be good to go!